msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase833517.java

```diff
@@ -0,0 +1,119 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.broker;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiSelector;
+
+import com.microsoft.identity.client.IAccount;
+import com.microsoft.identity.client.IMultipleAccountPublicClientApplication;
+import com.microsoft.identity.client.IPublicClientApplication;
+import com.microsoft.identity.client.ISingleAccountPublicClientApplication;
+import com.microsoft.identity.client.MultipleAccountPublicClientApplication;
+import com.microsoft.identity.client.Prompt;
+import com.microsoft.identity.client.PublicClientApplication;
+import com.microsoft.identity.client.SingleAccountPublicClientApplication;
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalAuthResult;
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalAuthTestParams;
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalSdk;
+import com.microsoft.identity.client.ui.automation.TestContext;
+import com.microsoft.identity.client.ui.automation.TokenRequestTimeout;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost;
+import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator;
+import com.microsoft.identity.client.ui.automation.constants.AuthScheme;
+import com.microsoft.identity.client.ui.automation.device.settings.GoogleSettings;
+import com.microsoft.identity.client.ui.automation.interaction.OnInteractionRequired;
+import com.microsoft.identity.client.ui.automation.interaction.PromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.AadPromptHandler;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment;
+import com.microsoft.identity.labapi.utilities.constants.ProtectionPolicy;
+import com.microsoft.identity.labapi.utilities.constants.PublicClient;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+import com.microsoft.identity.labapi.utilities.constants.UserRole;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+@SupportedBrokers(brokers = {BrokerMicrosoftAuthenticator.class})
+public class TestCase833517 extends AbstractMsalBrokerTest{
+
+    @Test
+    public void test_833517() throws Throwable{
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+        final UiDevice device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
```

Comment:
nit: I don't think you are actually using this object anywhere. Also, try to make sure no unnecessary imports are added to keep the code clean.